Ecosystem Partners/Marketplace Guides/getting-started-with-ruxit-blueprint.md

```diff
@@ -0,0 +1,144 @@
+
+### Solution Overview
+Ruxit provides the following:
+
+  * Java and JVM performance monitoring
+  * Node.js and Web framework monitoring
+  * Server and virtual infrastructure monitoring
+  * Docker monitoring
+  * Real-user response time in browsers
+  * JavaScript error detection in browsers
+  * Network monitoring at the process level
+  * Stack trace analysis
+  * Database throughput and response time analysis
+  * Intuitive touch-first dashboards
+  * Automatic root cause analysis with artificial intelligence
+  * No misleading alerts —you’re only notified of real problems
+  * .NET and IIS monitoring
+  * PHP monitoring
+  * Dynamic execution paths at the code level
+  * Database statement inspection
+  * Auto-detection of 3rd party performance impact
+  * Apdex ratings and user action counts by region
+  * Detection of end-user satisfaction levels
+  * Breakdowns by web page
+
+#### Benefits
+No alerting noise! Artificial Intelligence analysis and baselining
+Ruxit learns your environment and analyzes anomalies so you don’t have to. When something goes wrong, you get the solution—not just an alert. With the ability to discover all dependencies in your application stack, Ruxit delivers fast root-cause analysis.
+
+All-in-one solution. One agent covers everything
+The Ruxit all-in-one approach to monitoring combines user-experience-, network-, server-, and cloud-monitoring in a single agent. This makes deployment easy. Just install a single agent on each host. 
+
+No data sampling! Response times are measured in customers' browsers
+Ruxit automatically measures response times and detects JavaScript errors directly in your customers’ browsers. Ruxit even monitors the performance of 3rd party calls or applications that you rely on (for example, Facebook, Twitter, or CDNs).
+
+### Audience
+CenturyLink Cloud Users, Developers, Application Owners, Operations and Engineering 
+
+### Impact
+After reading this article, the user should feel comfortable getting started using the partner technology on CenturyLink Cloud.
+
+After executing the steps in this Getting Started document, the users will have a functioning Ruxit monitoring solution.
+
+### Offer
+Ruxit has provided a Blueprint that will easily add monitoring to your whole CenturyLink cloud environment and provide you the necessary performance and health answers of your environment.    
+
+### Prerequisite
+For running this Blueprint you first need to sign up for a free trial for Ruxit monitoring. 
+
+Please browse to [https://Ruxit.com](https://Ruxit.com) to set up an account.  You will receive an email with your account credentials and how to start monitoring with Ruxit.
+
+  ![Ruxit Image](../../images/ecosystem-ruxit-1.png)
+
+### Deploy Ruxit Blueprint
+Follow these step by step instructions to deploy Ruxit
```

Comment:
Made changes.  Thanks, Chris.  Hope that worked.  (There is no step after 7 though...)
Bob Stolzberg | Principal Engineer

CenturyLink() { your.link ("to what's next") }
Email: bob@ctl.io<mailto:bob@ctl.io>  M: (314) 397-2610  Schedule a Meeting<https://calendly.com/bobs>
CTL.io/Developers<http://www.ctl.io/developers> | @CenturyLinkSTL<https://twitter.com/CenturyLinkSTL> | Blog<http://www.ctl.io/blog> | Free Trial<https://www.ctl.io/free-trial/?largemargesentme>




From: Chris Little <notifications@github.com<mailto:notifications@github.com>>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Fri, 4 Dec 2015 13:04:34 -0800
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com<mailto:PublicKB@noreply.github.com>>
Cc: User <bob.stolzberg@centurylink.com<mailto:bob.stolzberg@centurylink.com>>
Subject: Re: [PublicKB] Urgent RuxIt Documentation (#675)


In Ecosystem Partners/Marketplace Guides/getting-started-with-ruxit-blueprint.md<https://github.com/CenturyLinkCloud/PublicKB/pull/675#discussion_r46732060>:

> +After reading this article, the user should feel comfortable getting started using the partner technology on CenturyLink Cloud.
> +
> +After executing the steps in this Getting Started document, the users will have a functioning Ruxit monitoring solution.
> +
> +### Offer
> +Ruxit has provided a Blueprint that will easily add monitoring to your whole CenturyLink cloud environment and provide you the necessary performance and health answers of your environment.
> +
> +### Prerequisite
> +For running this Blueprint you first need to sign up for a free trial for Ruxit monitoring.
> +
> +Please browse to [https://Ruxit.com](https://Ruxit.com) to set up an account.  You will receive an email with your account credentials and how to start monitoring with Ruxit.
> +
> +  ![Ruxit Image](../../images/ecosystem-ruxit-1.png)
> +
> +### Deploy Ruxit Blueprint
> +Follow these step by step instructions to deploy Ruxit


your numbering is broken in this section after step 10

-
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/675/files#r46732060>.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.